Client/Client.Qt/Settings.cpp

```diff
@@ -19,17 +19,23 @@ Settings& Settings::getInstance()
 void Settings::writeSettings()
 {
     settings->beginGroup("Font");
-    settings->setValue("FontSize", _fontSize);
+    settings->setValue("ChatFontSize", _fontSize);
     settings->endGroup();
 }
 
+void Settings::resetSettings()
+{
+    settings->clear();
+}
+
 void Settings::setFontSize(int size)
 {
     _fontSize = size;
 }
 
 int Settings::getFontSize()
```

Comment:
It is a good segment of code where you could try std::optional

Reply:
Also font size mut be >0